vnext/JSI/Shared/ChakraApi.h

```diff
@@ -0,0 +1,468 @@
+      return tls_exceptionThrower;
+    }
+
+   private:
+    static thread_local IExceptionThrower *tls_exceptionThrower;
+    IExceptionThrower *m_previous;
+  };
+
+  /**
+   * @brief Throws JavaScript exception with provided errorCode.
+   */
+  static void ThrowJsException(JsErrorCode errorCode);
+
+  /**
+   * @brief Throws native exception with provided message.
+   */
+  static void ThrowNativeException(char const *errorMessage);
+
+  /**
+   * @brief Adds a reference to a garbage collected object.
+   */
+  static uint32_t AddRef(JsRef ref);
+
+  /**
+   * @brief Releases a reference to a garbage collected object.
+   */
+  static uint32_t Release(JsRef ref);
+
+  /**
+   * @brief Creates a script context for running scripts.
+   */
+  static JsContextRef CreateContext(JsRuntimeHandle runtime);
+
+  /**
+   * @brief Gets the current script context on the thread..
+   */
+  JsContextRef GetCurrentContext();
+
+  /**
+   * @brief Sets the current script context on the thread.
+   */
+  void SetCurrentContext(JsContextRef context);
+
+  /**
+   * @brief Gets the property ID associated with the name.
+   */
+  static JsPropertyIdRef GetPropertyIdFromName(wchar_t const *name);
+
+  /**
+   * @brief Gets the property ID associated with the name.
+   */
+  static JsPropertyIdRef GetPropertyIdFromName(std::string_view name);
+
+  /**
+   * @brief Gets the property ID associated with the string.
+   */
+  static JsPropertyIdRef GetPropertyIdFromString(JsValueRef value);
+
+  /**
+   * @brief Gets the name associated with the property ID.
+   */
+  static wchar_t const *GetPropertyNameFromId(JsPropertyIdRef propertyId);
+
+  /**
+   * @brief Gets the string associated with the property ID.
+   */
+  static JsValueRef GetPropertyStringFromId(JsPropertyIdRef propertyId);
+
+  /**
+   * @brief Gets the symbol associated with the property ID.
+   */
+  static JsValueRef GetSymbolFromPropertyId(JsPropertyIdRef propertyId);
+
+  /**
+   * @brief Gets the type of property.
+   */
+  static JsPropertyIdType GetPropertyIdType(JsPropertyIdRef propertyId);
+
+  /**
+   * @brief Gets the property ID associated with the symbol.
+   */
+  static JsPropertyIdRef GetPropertyIdFromSymbol(JsValueRef symbol);
+
+  /**
+   * @brief Creates symbol and gets the property ID associated with the symbol.
+   */
+  static JsPropertyIdRef GetPropertyIdFromSymbol(std::wstring_view symbolDescription);
+
+  /**
+   * @brief Creates a JavaScript symbol.
+   */
+  static JsValueRef CreateSymbol(JsValueRef symbolDescription);
+
+  /**
+   * @brief Creates a JavaScript symbol.
+   */
+  static JsValueRef CreateSymbol(std::wstring_view symbolDescription);
+
+  /**
+   * @brief Gets the value of \c undefined in the current script context.
+   */
+  static JsValueRef GetUndefinedValue();
+
+  /**
+   * @brief Gets the value of \c null in the current script context.
+   */
+  static JsValueRef GetNullValue();
+
+  /**
+   * @brief Creates a Boolean value from a \c bool value.
```

Comment:
>\c [](start = 43, length = 2)

curious what does \c mean?

Reply:
It is to show the next word in monospace font. See [https://www.doxygen.nl/manual/commands.html#cmdc](https://www.doxygen.nl/manual/commands.html#cmdc)

---
In reply to: [470461924](https://github.com/microsoft/react-native-windows/pull/5710#discussion_r470461924) [](ancestors = 470461924)